packages/api/lib/writeRecords/write-granules.js

```diff
@@ -453,28 +459,33 @@ const _writeGranuleFiles = async ({
 const _writeGranuleRecords = async (params) => {
   const {
     postgresGranuleRecord,
-    apiGranuleRecord,
     knex,
     esClient = await Search.es(),
     granuleModel,
     executionCumulusId,
     granulePgModel,
     writeConstraints = true,
   } = params;
+  let { apiGranuleRecord } = params;
   let pgGranule;
+  let pgGranuleResult;
 
   log.info('About to write granule record %j to PostgreSQL', postgresGranuleRecord);
   log.info('About to write granule record %j to DynamoDB', apiGranuleRecord);
 
   try {
     await createRejectableTransaction(knex, async (trx) => {
-      pgGranule = await _writePostgresGranuleViaTransaction({
+      pgGranuleResult = await _writePostgresGranuleViaTransaction({
         granuleRecord: postgresGranuleRecord,
         executionCumulusId,
         trx,
         granulePgModel,
         writeConstraints,
       });
+      if (pgGranuleResult.status === 'failed') {
```

Comment:
🐜/nit  -- consider a status something more like 'dropped'.    It wasn't obvious when first talking about this ticket, but there's a distinct meaning in failure (knex *throws*) versus what's happening here, and that might lead people to the wrong conclusion.   Given the code complexity, it's worth considering.

Reply:
changed to dropped.